android/src/main/java/com/ahm/capacitor/camera/preview/CameraActivity.java

```diff
@@ -247,12 +242,9 @@ public void takePicture(final int width, final int height, final int quality) th
         stillCaptureRequestBuilder.addTarget(reader.getSurface());
         stillCaptureRequestBuilder.set(CaptureRequest.CONTROL_MODE, CameraMetadata.CONTROL_MODE_AUTO);
 
-        if (!disableExifHeaderStripping) {
```

Comment:
why have disableExifHeaderStripping check been removed? will it introduce a breaking change?